site/src/testHelpers/entities.ts

```diff
@@ -1339,7 +1343,7 @@ export const MockEntitlementsWithScheduling: TypesGen.Entitlements = {
   }),
 }
 
-export const MockExperiments: TypesGen.Experiment[] = []
+export const MockExperiments: TypesGen.Experiment[] = ["workspace_actions"]
```

Comment:
@BrunoQuaresma I gave up on the testing issue we chatted about and just adjusted the experiment mock. This should be okay for now because no other tests are using it, as far as I can tell. Open to suggestions here.

Reply:
I'm good with that!